pkg/base1/cockpit-components-listing.jsx

```diff
@@ -128,7 +129,7 @@ define([
             var count_display = null;
 
             var header_entries = this.props.columns.map(function(itm) {
-                if (typeof itm === 'string' || itm instanceof String || React.isValidElement(itm))
+                if (typeof itm === 'string' || itm === undefined || itm instanceof String || React.isValidElement(itm))
```

Comment:
I encountered the same problem when working on the docker stuff, and I decided against this patch. I think it will be hard to catch bugs in the future if we accept `undefined` here.